components/ItemComponents/Content/MainMetadata.js

```diff
@@ -40,6 +40,24 @@ class MainMetadata extends React.Component {
     this.setState({ isOpen: true });
   }
 
+  renderRightsBadge = (item) => {
+    /* 
+    for situations where the rights are in sourceResource
+    see: https://dp.la/item/7f2973c3c4429087b4874725f3bc67ad
+    items should not have multiple rights but showing them in case a proper uri is present
+    */
+
+    if (item.rights && Array.isArray(item.rights)) {
+      item.rights.map((theRight, index) => {
+        return <RightsBadge url={theRight} key={index} />;
+      })
+    } else if (item.rights) {
+      return <RightsBadge url={item.rights} />
+    } else if (item.edmRights) {
+      return <RightsBadge url={item.edmRights} />
```

Comment:
I think we want edmRights to take precedence here, which seems like would require this `if` statement being flipped around... am I right? (ha ha ha)